pkg/controller/v1alpha2/applicationconfiguration/applicationconfiguration.go

```diff
@@ -453,7 +457,10 @@ func (w Workload) Status() v1alpha2.WorkloadStatus {
 		Traits: make([]v1alpha2.WorkloadTrait, len(w.Traits)),
 		Scopes: make([]v1alpha2.WorkloadScope, len(w.Scopes)),
 	}
-	for i := range w.Traits {
+	for i, tr := range w.Traits {
+		if tr.Definition.Name == util.Dummy && tr.Definition.Spec.Reference.Name == util.Dummy {
```

Comment:
It seems that this only applies to traits. What about workloads?

Reply:
we don't check workloadDef when apply worklaod.